src/urbit/lib/chess.hoon

```diff
@@ -382,12 +386,16 @@
     ?:  =(^piece piece)
       `square
     ~
+  ::
+  ::  check whether a square is occupied and return the side occupying the square
   ++  occupied
     |=  square=chess-square
     ^-  (unit chess-side)
     =/  piece  (~(get by board) square)
     ?~  piece  ~
     [~ -.u.piece]
+  ::
+  ::  move a piece by deleting from previous location and adding to new location, promoting if needed
```

Comment:
I don't think it's in the style guide, but I believe Hoon style is to break comments at the 80 character line, so this comment would look like:
```
:: move a piece by deleting from previous location and adding to new
:: location, promoting if needed
```